lib/core/src/lib/card-view/components/card-view/card-view.component.ts

```diff
@@ -57,4 +57,8 @@ export class CardViewComponent {
     /** String separator between multi-value property items. */
     @Input()
     multiValueSeparator: string = DEFAULT_SEPARATOR;
+
+    /** Toggles whether or not to show label for multivalued chip property. */
+    @Input()
+    displayLabelForChips: boolean = true;
```

Comment:
I'm not sure I understand, but how it is backward compatible this way? Didn't we say that we would keep the old behaviour as default?

Reply:
Done.